src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
```

Comment:
Link to a SCSS file in HTML is incorrect. Browsers do not understand SCSS files directly. You should compile your SCSS to CSS and link to the CSS file instead.